setup/resources/Microsoft.FSharp.NetSdk.Shim.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <Import Project="$(MSBuildThisFileDirectory)..\..\..\..\..\Common7\IDE\CommonExtensions\Microsoft\FSharpCompiler\Microsoft.FSharp.NetSdk.props" />
+  <Import Project="$(VsInstallRoot)\MSBuild\15.0\Bin\FSharp\Microsoft.FSharp.NetSdk.props" />
```

Comment:
Can `VsInstallRoot` be counted on to exist?

Reply:
According to [this](https://msdn.microsoft.com/en-us/library/c02as0cs.aspx) that value will always be present, but reading the comment it could behave oddly with `$(PlatformToolset)`.  Would you prefer I go back to the `$(MSBuildThisFileDirectory)..\..\..\etc` method?  It doesn't look as clean, but will always produce consistent results.